tardis/io/atom_data/util.py

```diff
@@ -51,3 +51,32 @@ def resolve_atom_data_fname(fname):
         f"Atom Data {fname} is not found in current path or in TARDIS data repo. {atom_data_name} "
         "is also not a standard known TARDIS atom dataset."
     )
+
+
+def set_atom_data_attributes(atom_data, store, attribute):
```

Comment:
This seems a little broad, maybe some sort of schema for the AtomData is worthwhile?

Reply:
This was some refactoring to clean up a lot of try-except blocks from the HDF reader classmethod. This would certainly be replaced if a schema was implemented.